tests/logger.py

```diff
@@ -2,14 +2,17 @@
 import time
 import sys
 
-LOG_FOLDER      = "logs"
-LOG_EXTENSION   = ".log"
-LOG_DELIMITER   = "| "
+LOG_FOLDER = "logs"
+LOG_EXTENSION = ".log"
+LOG_DELIMITER = "| "
+
 
 class Logger():
-    def __init__(self, fname):
+    def __init__(self, fname, mode):
         self.terminal = sys.stdout
-        #types of messages for logging
+        self.mode = mode
```

Comment:
What does ```mode``` do here?
I do not understand why we should open the logfile from write only if mode is set